src/server/game/Entities/Player/Player.cpp

```diff
@@ -18026,8 +18033,11 @@ void Player::_LoadAuras(PreparedQueryResult result, uint32 timediff)
                 continue;
             }
 
+            const ChrRacesEntry* raceEntry = sChrRacesStore.LookupEntry(GetRace());
+            ASSERT_NOTNULL(raceEntry);
```

Comment:
Same here, const placement and use AssertEntry

Reply:
Okie, I think I changed it correctly now.